hpx/runtime/parcelset/parcelport_impl.hpp

```diff
@@ -184,6 +191,49 @@ namespace hpx { namespace parcelset
 
         }
 
+    protected:
+        void put_parcel_await(
+            locality const & dest, parcel p, write_handler_type f, bool trigger
```

Comment:
I noticed that, in hindsight it was always `true` only. I left it as I thought you were planning for it to be available for any of the upcoming parcel port implementations. But I'm fine to remove it.